she_codes_news/news/forms.py

```diff
@@ -0,0 +1,16 @@
+from django import forms
+from django.forms import ModelForm
+from .models import NewsStory, Category
+
+
+class StoryForm(ModelForm):
+    class Meta:
+        model = NewsStory
+        fields = ['title', 'pub_date', 'category','image','content'] #we removed author
+        widgets = {
+            'title': forms.TextInput(attrs={'class': 'form-field-test'}),      #added title line to try connect CSS
+            'pub_date': forms.DateInput(format=('%d/%m/%Y'), attrs={'class':'form-control', 'placeholder':'Select a date','type':'date'}),
+            'category': forms.Select(attrs={'class': 'form-field-test'}),
+            'image': forms.URLInput(attrs={'class': 'form-field-test'}),
```

Comment:
there's a lot of form-field-test here, you could have just over-ridden all of the forms fields in one line, rather than making everything a class